src/main/typescript/wcardinal/ui/d-slider-value.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2019 Toshiba Corporation
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { DBaseState } from "./d-base-state";
+import { DTextBase, DTextBaseOptions, DThemeTextBase } from "./d-text-base";
+
+export interface DSliderValueOptions<
+	VALUE = number,
+	THEME extends DThemeSliderValue = DThemeSliderValue
+> extends DTextBaseOptions<VALUE, THEME> {
+	value?: number;
+	precision?: number;
+	rounder?( value: number ): number;
```

Comment:
In the case of the `rounder?:( value: number ): number;`, need to call as if it is a method of the DSliderValueOptions (`options.rounder( value )`).
```suggestion
	rounder?: ( value: number ) => number;
```